packages/editor/src/utils/position.ts

```diff
@@ -46,9 +46,13 @@ export type PositionOptions = {
   parent?: HTMLElement | Element;
 };
 export function getPosition(
-  element: HTMLElement,
+  node: Node,
   options: PositionOptions
 ): { top: number; left: number } {
+  const element = node.parentElement;
+  if (element === null) {
+    throw new Error(`Unsupported node type ${node.nodeType}`);
+  }
```

Comment:
Awesome! However, we aren't always looking for the parentElement or need that so the check should me just on the element. Or wait.

I think we can do a check like this:

1. Check if the `node` is an instance of `HTMLElement`
2. If it is, just use that.
3. If it isn't:
    1. We can either just return
    2. Or use `node.parentElement` (in cases where it's a TextNode).

Reply:
Sounds good, adding now.

Reply:
6b32274fc93e621e31cc7ada2d9fd94b0a09649a